src/main/java/frc/robot/subsystems/ShooterPivot.java

```diff
@@ -27,6 +27,7 @@ public class ShooterPivot extends TrapezoidProfileSubsystem {
     // As of writing the above note we still may want to change the limits
     public static final double ANGLE_TOLERANCE_RADIAN = Math.toRadians(3.0); //TODO: set for 2024
 
+    public static final double OVERRIDE_RADIANS = Math.toRadians(5.0);
```

Comment:
This is not used. Also, 5 degrees is very big. The shoot angle varies between ~20 and 35 degrees total.